docs/api/tests.foreman.ui.rst

```diff
@@ -19,6 +19,13 @@
     :members:
     :undoc-members:
 
+:mod:`tests.foreman.ui.test_capsule`
+--------------------------------------------
```

Comment:
nitpick: can this line be trimmed down? No worries if you have other things on your plate and want to get this merged, as this is a small thing.